brave_paks.gni

```diff
@@ -77,6 +77,7 @@ template("brave_extra_paks") {
       "//brave/browser/resources/settings:resources",
       "//brave/components/brave_sync:resources",
       "//brave/ui/webui/resources",
+      "//brave/vendor/bat-native-ads/resources",
```

Comment:
this should be wrapped in `brave_ads_enabled` along with whatever pak file it produces

Reply:
Mean we merge this to fix master and then I will fix when I return?